manifests/reposync-crd.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 Google LLC
+# Copyright 2023 Google LLC
```

Comment:
Do all of the headers need to be updated?

Reply:
Nope. Only the CRD files because they're regenerated in the presubmit job.